prologin/pages/templates/pages/about-privacy-policy.html

```diff
@@ -0,0 +1,184 @@
+                    Prologin ;</li>
+                <li>d'adapter les repas aux contraintes alimentaires de chacun ;</li>
+                <li>de répondre aux sollicitations de la part des participants ;</li>
+                <li>de piloter l'activité de l'association (évaluation de la
+                    satisfaction des participants, production de statistiques).</li>
+            </ul>
+        </p>
+
+        <h4>Base légale</h4>
+        <p>Ce traitement repose sur le consentement des inscrits et de leur
+            responsable légal le cas échéant (mineurs de 16 ans), ce sont les
+            articles 6(1)a et 8 du RGPD.</p>
+    </div>
+
+    <button class="btn btn-default btn-block" type="button"
+        data-toggle="collapse" data-target="#processed_data"
+        aria-expanded="false" aria-controls="processed_data">
+        <h3>Données traitées</h3>
+    </button>
+    <div class="collapse" id="processed_data">
+        <h4>Catégories de données traitées</h4>
+        <p>L'inscription sur le site nécessite les catégories de données
+            suivantes :</p>
+        <ul>
+            <li>Données d'identification : pseudonyme, adresse email ;</li>
+            <li>Internet : mesure d'audience ;</li>
+        </ul>
+        <p>La participation au concours requiert des informations
+            complémentaires :</p>
+        <ul>
+            <li>Données d'identification : nom, prénom, adresse postale, date de
+                naissance... ;</li>
+            <li>Vie scolaire (dite professionnelle) : niveau d'étude ;</li>
+            <li>Vie personnelle : habitudes alimentaires ;</li>
+            <li>Internet : mesure d'audience ;</li>
+            <li>Données de santé : éventuels traitements médicaux et allergies.</li>
+        </ul>
+
+        <h4>Source des données</h4>
+        <p>Ces informations sont recueillies directement auprès du participant,
+            et/ou de son responsable légal le cas échéant (mineurs de 16 ans).</p>
+        <h4>Caractère obligatoire des réponses</h4>
+        <p>L'inscription au site Prologin prévoit, sauf mention contraire, le
+            recueil obligatoire de données qui sont nécessaires au traitement de
+            l'inscription.</p>
+        <p>La participation à une édition du concours Prologin prévoit, sauf
+            mention contraire, le recueil obligatoire de données qui sont
+            nécessaires au traitement de l'inscription et l'accueil sur site
+            dans de bonnes conditions.</p>
+
+        <h4>Prise de décision automatisée</h4>
+        <p>Le traitement peut inclure une décision entièrement automatisée : les
+            demandes de participation au concours Prologin d'un niveau
+            manifestement insuffisant (évaluation par le juge en ligne Prologin)
+            sont automatiquement écartées. Bien que cette décision automatique
+            ne produise ni effet juridique ni impact significatif, Prologin peut
+            être contacté directement </p>
```

Comment:
Missing text at the end:
```suggestion
            être <a href="mailto:info@prologin.org">contacté directement</a> pour demander une évaluation manuelle.  </p>
```